curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-asynchronous-programming-by-building-an-fcc-forum-leaderboard/643ee973c162ae017b3d30db.md

```diff
@@ -1,22 +1,22 @@
 ---
 id: 643ee973c162ae017b3d30db
-title: Step 23
+title: Step 22
 challengeType: 0
-dashedName: step-23
+dashedName: step-22
 ---
 
 # --description--
 
-To show some data in the Activity column, you need to use the `bumped_at` property of each topic. This property is a timestamp which is in the ISO 8601 format with the timezone information. You need to process this data before displaying it on the project. 
+To show some data in the `Activity` column, you need to use the `bumped_at` property of each topic, which is a timestamp in the ISO 8601 format. You need to process this data before you can display how much time has passed since a topic saw any activity.
```

Comment:
```suggestion
To display data in the `Activity` column, you need to use the `bumped_at` property of each topic, which is a timestamp in the ISO 8601 format. You need to process this data before you can show how much time has passed since a topic had any activity.
```